lib/components/Newsletter.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class Newsletter extends StatefulWidget {
```

Comment:
The field doesn't override an inherited getter or setter • lib/components/Newsletter.dart:11:9 • override_on_non_overriding_member

Fix this issue